packages/headless/src/features/commerce/recommendations/recommendations-state.ts

```diff
@@ -0,0 +1,22 @@
+import {CommerceAPIErrorStatusResponse} from '../../../api/commerce/commerce-api-error-response';
+import {Product} from '../../../api/commerce/common/product';
+
+export interface RecommendationsState {
+  slotId: string;
+  headline: string;
+  error: CommerceAPIErrorStatusResponse | null;
+  isLoading: boolean;
+  requestId: string;
```

Comment:
I don't see us using the `requestId` property. I think we can remove it. This will affect the `fetchRecommendations.pending` case in the slice as well.
```suggestion
```

Reply:
Good point